notebooks/azureml_model_headers_demo/german_credit_azure_ml_certifai_pro_demo.ipynb

```diff
@@ -0,0 +1,1130 @@
+    "                            predict_endpoint=service_svm_uri)\n",
+    "scan.add_model(first_model)\n",
+    "\n",
+    "second_model = CertifaiModel('logistic',\n",
+    "                            predict_endpoint=service_logistic_uri)\n",
+    "scan.add_model(second_model)\n",
+    "\n",
+    "# Add corresponding model headers for service authentication and content-type\n",
+    "\n",
+    "# add the default headers applicable to all models\n",
+    "scan.add_model_header(header_name='Content-Type',header_value='application/json')\n",
+    "\n",
+    "# add defined headers corresponding to auth keys for respective model services\n",
+    "scan.add_model_header(header_name='Authorization', header_value=f'Bearer {service_svm_keys[0]}', model_id='SVM')\n",
+    "scan.add_model_header(header_name='Authorization', header_value=f'Bearer {service_logistic_keys[0]}', model_id='logistic')\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### add the evaluation dataset to scan object\n",
+    "\n",
+    "- `evaluation dataset` dataset to be used by cortex certifai to evaluate the model against"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an evaluation object and pass the evaluation dataset(csv) here \n",
+    "eval_dataset = CertifaiDataset('evaluation',\n",
+    "                               CertifaiDatasetSource.csv(url=f'abfs://{az_container_name}/{az_german_credit_blob_name}'))\n",
+    "scan.add_dataset(eval_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### evaluating model fairness \n",
+    "\n",
+    "- add `fairness` as evaluation type to scan object\n",
+    "- create an `evaluation_dataset_id` to refer to added evaluation datset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup an evaluation for fairness on the above dataset using the model\n",
+    "# We'll look at disparity between groups defined by marital status and age\n",
+    "scan.add_fairness_grouping_feature(CertifaiGroupingFeature('age'))\n",
+    "scan.add_fairness_grouping_feature(CertifaiGroupingFeature('status'))\n",
+    "scan.add_evaluation_type('fairness')\n",
+    "scan.evaluation_dataset_id = 'evaluation'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Because the dataset contains a ground truth outcome column which the model does not\n",
+    "# expect to receive as input we need to state that in the dataset schema (since it cannot\n",
+    "# be inferred from the CSV)\n",
+    "scan.dataset_schema.outcome_feature_name = 'outcome'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
think things are out of order. You're getting the logs before running it

Reply:
fixed